### ISSUE TITLE
Pricing of mails originating from Lambda functions

### ISSUE THREAD

**Issue by paulbusse** [dwyl/aws-ses-lambda#21]
Hi,

I was just wondering how AWS is invoicing SES mails sent from Lambda functions?
SES has two types of invoices: 
1. If the mail is originating from EC2 instances the first 62.000 mails are free
2. In other cases  you pay $0.10 for a 1000 mails
So, under which of the 2 cases fall mails send from Lambda functions

Regards

Paul

**Comment by nelsonic** [comment]
Hi Paul, thank you for opening this issue it will no doubt be a question others have. 👍 
Our purpose with having a `Lambda` function to handle all our email is simple: never having to _think_ about it. 

Sadly, I can confirm that the `EC2` "first 65k emails" free usage tier does **_not_ apply** to **AWS Lambda**. 😞 
Here's a screenshot of the billing on our `aws-ses-lambda` "test" account showing the only thing we are paying for is **`SES`**:
![image](https://user-images.githubusercontent.com/194400/88037982-84391b00-cb3d-11ea-994f-13dd87fa5c49.png)
The first 1 Million `Lambda` executions and 100k `SNS` calls are free so the only thing we are paying for is SES. 

If you are _already_ paying for an `EC2` instance, then it's worth using that for sending/receiving your email _especially_ if you are sending tens of thousands of emails per month. If you have an established account and you are thinking of using a dedicated `EC2` instance to send/receive email you will need to do the cost-benefit analysis of maintaining a server. 

https://aws.amazon.com/ses/pricing
![image](https://user-images.githubusercontent.com/194400/88038321-f873be80-cb3d-11ea-9c51-bce034aa7a4b.png)

SES is in the "Always Free" category, so if you stay below the 65k, it will _always_ be free (from an `EC2` instance).
![image](https://user-images.githubusercontent.com/194400/88039973-2bb74d00-cb40-11ea-974a-b2d234f9b8cb.png)

> For people signing up for a **`new` AWS account**, the **`EC2` Free Tier** (12 Months), 
**`t2.micro`** for 750h/month will be a good way to run an app and get free email. 

So to recap: if you _already_ have an `EC2` instance, use it to send email. 
If you don't have an `EC2` instance and would have to set one up just to send email, 
Lambda's ("always free tier") will be cheaper and less hassle if you only send a few thousand emails per month.

We aren't using any `EC2` instances because the TCO is higher than DigitalOcean so even if we got **$6.50** worth of "free" email credit per month, we would still be paying more than that for our `EC2` + `EBS`, `Elastic IP`, etc. 

Hope this is helpful. ☀️

**Comment by paulbusse** [comment]
That is more than I asked for :-) thanks a lot.

**Comment by nelsonic** [comment]
Hi @paulbusse, I'm _very_ curious what you decide to do. 💭 
At present our email stats aggregator https://github.com/dwyl/email is running on Heroku (Free Tier).
It might make sense to switch it over to AWS `EC2` in the future. 
Are you thinking of building your own email system on top of `SES`?
